### ISSUE TITLE
Computer Vision: Facebook Detectron 2 object detection algorithms

### ISSUE THREAD

**Issue by ClaudeSChretien** [BrisPy/talks#52]
Computer vision was a very popular topic when I did my internship at the Canadian defense back in 2018. It has continue to improve since. I just saw that a new version of the powerful computer vision library from Facebook got released.

https://github.com/facebookresearch/detectron2

I don't know if you already have any talk regarding computer vision, but it might be interesting.
The talk can cover different topics:
- Real time algorithms. is 2s per image is enough?
- Different level of detection (Classification, localisation, segmentation)
- How to install Detectron.
- Possibilities with the library.
- Hardware available (Nvidia Jetson, Xavier etc..)

Thanks!

**Comment by kaiw** [comment]
I don't think we've had a computer vision talk, certainly not recently! I think this definitely sounds interesting.

Are you offering to present this yourself?

**Comment by ClaudeSChretien** [comment]
Great! I would really like to give the talk about this topic. What would be the next steps? Do you need a summary of the presentation?
Thanks! :)

**Comment by kaiw** [comment]
No, we just need a talk title, and to figure out dates where you might be prepared to give the talk!

We currently have a slot to fill on Wednesday February 12th, if you think you could be ready by then?

**Comment by ClaudeSChretien** [comment]
Hi Kaiw,
That's perfect for me for February 12th. How much time do I have?
For the title, what about: 
- Detectron2, the Facebook open source computer vision platform.
Might be too specific, feel free to modify it.
Thanks! :)

**Comment by kaiw** [comment]
Great! I've updated https://github.com/BrisPy/talks/blob/master/README.md with your talk, so you'll be in the second slot.

As for time, we usually target talks at *about* 25 minutes + 5 minutes questions, but if you're a little over or under then that's also fine.

**Comment by ClaudeSChretien** [comment]
That's perfect for me.
Thanks! :)
